<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>904</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="sifrovaniGUI">
      <property name="text">
       <string>Zadejte text pro šifrování</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEditTextDesifrovani"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="desifrovaniGUI">
      <property name="text">
       <string>Zadejte text pro dešifrování</string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QPushButton" name="pushZasifrovat">
      <property name="text">
       <string>Zašifrovat</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="zasifrovanyTextGUI">
      <property name="text">
       <string>Zašifrovaný text: </string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEditTextSifrovani"/>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QPushButton" name="pushDesifrovat">
      <property name="text">
       <string>Dešifrovat</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QLabel" name="desifrovanyTextGUI">
      <property name="text">
       <string>Dešifrovaný text: </string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="3">
     <widget class="QLabel" name="vyfiltrovanyTextGUI">
      <property name="text">
       <string>Vyfiltrovaný text: </string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0" colspan="3">
     <widget class="QLabel" name="abecedaGUI">
      <property name="text">
       <string>Abeceda: </string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QLabel" name="sifrovaciAbecedaGUI">
      <property name="text">
       <string>Šifrovací abeceda: </string>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditCisloB"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditCisloA"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="cisloB">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="cisloA">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
